### DISCUSSION TITLE
Change colour, opacity and border of footprint?

### DISCUSSION THREAD

**Discussion by NeighNeighNeigh** [rxhanson/RectanglePro-Community#233] [Q&A]
Back in Hookshot days I recall being able to set custom settings for the appearance of the footprint. This was done through Terminal. I can't seem to figure out how to do this in Rectangle Pro, can anyone help?

**Comment by rxhanson** [comment]
You can use the terminal commands for Rectangle: https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#modify-the-footprint-displayed-for-drag-to-snap-area

Just replace `com.knollsoft.Rectangle` with `com.knollsoft.Hookshot`.

Restart the app after executing a terminal command. 

Here they are for quick reference:

> ## Modify the "footprint" displayed for drag to snap area
> 
> Adjust the alpha (transparency). Default is 0.3.
> 
> ```bash
> defaults write com.knollsoft.Hookshot footprintAlpha -float <VALUE_BETWEEN_0_&_1>
> ```
> 
> Change the border width. Default is 2 (used to be 1).
> 
> ```bash
> defaults write com.knollsoft.Hookshot footprintBorderWidth -float <NUM_PIXELS>
> ```
> 
> Disable the fade.
> 
> ```bash
> defaults write com.knollsoft.Hookshot footprintFade -int 2
> ```
> 
> Change the color.
> 
> ```bash
> defaults write com.knollsoft.Hookshot footprintColor -string "{\"red\":0,\"blue\":0.5,\"green\":0.5}"
> ```
> 
> Change the animation duration. The value is a multiplier. Default is 0 (no animation).
> 
> ```bash
> defaults write com.knollsoft.Hookshot footprintAnimationDurationMultiplier -float <MULTIPLIER>
> ```
